### ISSUE TITLE
iOS 9.3.3 breaks hot reloding

### ISSUE THREAD

**Issue by funkjunky** [facebook/react-native#9226]
Changes are no longer reflected, most buttons are now missing from dev menu.

**Comment by funkjunky** [comment]
This issue may be related to https://github.com/facebook/react-native/issues/8419

**Comment by Dean177** [comment]
What is being assigned to `jsCodeLocation` in `AppDelagate.m`?

**Comment by funkjunky** [comment]
`jsCodeLocation = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index.ios" fallbackResource:nil];`

**Comment by Dean177** [comment]
Change it to: `
#ifdef DEBUG
    jsCodeLocation = [NSURL URLWithString:@"http://PUT_YOUR_IP_HERE.2:8081/index.ios.bundle?platform=ios&dev=true"];
  #else
    jsCodeLocation = [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index.ios" fallbackResource:nil];
  #endif
`
Redploy and all will be well

**Comment by funkjunky** [comment]
Unfortunately that didn't work, but I tried modifying the original line adding .bundle?platform=... and that did the trick. I noticed when it was loading it had some xib or something in the url, which probably caused the normal ip address to not work.

Regardless it works with all the extra suffixes :) Thanks.

I still think this is a bug though. React Native no longer works out of the box with an iphone that has 9.3.3 installed.
